azure-iot-device/azure/iot/device/iothub/transport/mqtt/mqtt_topic.py

```diff
@@ -252,3 +252,38 @@ def encode_properties(message_to_send, topic):
         topic += user_properties_encoded
 
     return topic
+
+
+def get_twin_response_topic_for_subscribe():
+    return "$iothub/twin/res/#"
```

Comment:
>$iothub/twin [](start = 12, length = 12)

can this be a topic base for twin like we have a topic base for iothub telemetry operations ?

Reply:
That's a good idea, but I think that would make the code a little harder to read and the same value isn't there (because the base is constant, unlike telemetry which is based on device_id and module_id).

---
In reply to: [287524645](https://github.com/Azure/azure-iot-sdk-python-preview/pull/96#discussion_r287524645) [](ancestors = 287524645)